src/api/api.cpp

```diff
@@ -320,6 +316,16 @@ struct migraphx_onnx_options
     migraphx::onnx_options object;
 };
 
+extern "C" struct migraphx_file_options;
```

Comment:
How about the rename this to serialization_option? @pfultz2

Reply:
Lets just keep the name the same.